src/polycap-capil.c

```diff
@@ -474,9 +475,15 @@ STATIC int polycap_capil_reflect(polycap_photon *photon, double alfa, polycap_ve
 				free(w_leak);
 				return -1;
 			}
+			n_shells = round(sqrt(12. * phot_temp->description->n_cap - 3.)/6.-0.5);
 			for(i=0; i<=description->profile->nmax; i++){
-				capx_temp[i] = description->profile->ext[i] * capx_cntr / description->profile->ext[0];
-				capy_temp[i] = description->profile->ext[i] * capy_cntr / description->profile->ext[0];
+				if(n_shells == 0.){ //monocapillary case, normally code should never reach here (wall_trace should not return 1 for monocaps)
```

Comment:
Misschien een domme opmerking maar moet voor een monocapillair `n_shells` niet gelijk zijn aan 1?

Reply:
Het is eigenlijk meer het maximale nr/id van de shells, ipv het echt aantal shells.
Dus 1 shell (1 capillair) geeft n_shells = 0, zodra je meer dan 1 capillair hebt is n_shells 1 of meer

Is idd mss niet super inuitief, maar de meeste gebruikers gaan hier niet moeten aankomen, dus laat ik het maar zo :)